custom_components/yamaha_ynca/media_player.py

```
@@ -44,7 +44,7 @@
 from .input_helpers import InputHelper
 
 if TYPE_CHECKING:  # pragma: no cover
-    from ynca.subunits.zone import ZoneBase, Main
+    from ynca.subunits.zone import Main, ZoneBase
```

Ruff (0.8.2)

Move import `ynca.subunits.zone.Main` out of type-checking block. Import is used for more than type hinting.

Move out of type-checking block

(TC004)